src/main/kotlin/org/jitsi/nlj/rtp/TransportCcEngine.kt

```diff
@@ -121,6 +121,8 @@ class TransportCcEngine(
         }
 
         for (receivedPacket in tccPacket) {
+            if (!receivedPacket.received)
+                continue
```

Comment:
why was it important to track unreceived packets there if the only thing we do is skip over them in the loop?

Reply:
I haven't yet checked in the code which uses this.  Coming soon!